staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types.go

```diff
@@ -193,7 +190,7 @@ type JSONSchemaProps struct {
 }
 
 type JSON struct {
-	Raw []byte `protobuf:"bytes,1,opt,name=raw"`
+	Raw []byte `json:"raw" protobuf:"bytes,1,opt,name=raw"`
```

Comment:
Is that json tag needed? We implement JSON marshalling manually.

Reply:
will fix.